src/main/java/com/redhat/cloud/notifications/RestAction.java

```diff
@@ -50,19 +54,31 @@ public class RestAction {
     @ISO8601Timestamp
     public String timestamp;
 
-    @javax.validation.constraints.Positive
+    @Positive
```

Comment:
For better maintainability:
```suggestion
    // This field can only contain a positive number. We use the String type to allow leading zeros in the value.
    @Positive
```